api/src/main/java/io/druid/data/input/impl/FloatDimensionSchema.java

```diff
@@ -27,10 +27,16 @@
 {
   @JsonCreator
   public FloatDimensionSchema(
-      @JsonProperty("name") String name
+      @JsonProperty("name") String name,
+      @JsonProperty("multiValueHandling") MultiValueHandling multiValueHandling
```

Comment:
multi-value Float columns don't exist, so may as well simplify this and always pass `super(name, null);`